_data/simple-icons.json

```
@@ -6044,6 +6044,11 @@
             "hex": "25313C",
             "source": "https://www.iris.co.uk"
         },
+        {
+            "title": "ISC2",
```

```suggestion
            "title": "(ISC)²",
```

We should always follow the spell of their trademark.

---

Update the PR as per your suggestion.